package.json

```diff
@@ -80,6 +80,6 @@
   },
   "packageManager": "pnpm@8.15.2",
   "engines": {
-    "node": "^14.18.0 || >=16.10.0"
+    "node": "^16.10.0 || >=18.0.0"
```

Comment:
### Note

This update is based on the following issue and comments.

- https://github.com/nuxt/nuxt/pull/23821
- https://github.com/nuxt/nuxt/pull/23821#pullrequestreview-1689862898

Reply:
Looks good but would you mind to split this PR? 🙏🏼

Reply:
OK 🙆
This content has been split into a PR here.

- nuxt/cli#356